core/src/main/scala/org/bitcoins/core/protocol/tlv/TLV.scala

```diff
@@ -278,16 +284,31 @@ object EnumEventDescriptorV0TLV extends TLVFactory[EnumEventDescriptorV0TLV] {
     require(count.toInt == result.size,
             "Did not parse the expected number of outcomes")
 
-    EnumEventDescriptorV0TLV(result)
+    EnumEventDescriptorV0TLV(nonce, result)
   }
 }
 
-case class RangeEventDescriptorV0TLV(start: Int32, stop: Int32, step: UInt16)
+case class RangeEventDescriptorV0TLV(
+    nonce: SchnorrNonce,
+    start: Int32,
+    count: UInt32,
+    step: UInt16)
```

Comment:
So what happens when `count` and `step` are conflicting? I.e.

```scala

val count = 1
val step = 5

val start = 1
val stop = 15
```

This is internally inconsistent no?

Reply:
There is no more `stop` in the spec

Reply:
:man_facepalming: , i somehow put "stop" in there magically in my head :P.